### ISSUE TITLE
Format: css styles snake_case -> camelCase

### ISSUE THREAD

**Issue by gylman** [orkhan-m/product-feedback-app-react#3]
In React projects, when using `module.css`, **camelCase** is generally the recommended naming convention for CSS class names. Here's why:

1. **JavaScript Compatibility**: When you use CSS Modules, the classes are often imported as JavaScript objects, and since JavaScript uses camelCase for object properties, it's more natural to follow that convention. For example, when using CSS Modules, you typically access the class name like this:

   ```jsx
   import styles from './Component.module.css';

   function MyComponent() {
     return <div className={styles.myClassName}>Hello</div>;
   }
   ```

   In this example, `myClassName` is camelCase, and it aligns with JavaScript's property naming conventions.

2. **Consistency**: Since React components themselves typically follow camelCase conventions (e.g., `MyComponent`, `myFunction`), keeping your class names in camelCase helps maintain consistency throughout your project.

3. **Tooling and Best Practices**: Most modern JavaScript and React codebases, linters, and style guides (like Airbnb's or Google's style guide) tend to favor camelCase for CSS class names when working in a JavaScript ecosystem.

### Example with CSS Module using camelCase:
```css
/* Component.module.css */
.container {
  background-color: lightblue;
}

.headerText {
  color: darkblue;
}
```

### Example JSX:
```jsx
import styles from './Component.module.css';

function MyComponent() {
  return (
    <div className={styles.container}>
      <h1 className={styles.headerText}>Hello, World!</h1>
    </div>
  );
}
```

### When to Use Snake Case?
- **Consistency with Existing Code**: If you're working in a project or with a team that has a pre-existing convention of using snake_case, then it's fine to continue that for consistency.
- **Global CSS**: In projects that don't use CSS Modules (e.g., for global CSS or older codebases), snake_case or kebab-case is more common, since that's a typical convention for CSS.

In summary, for CSS Modules in React, **camelCase** is the preferred choice.

**Comment by orkhan-m** [comment]
fix has been merged to the main branch
